blocks/library/text/index.js

```diff
@@ -3,6 +3,7 @@
  */
 import { __ } from 'i18n';
 import { concatChildren } from 'element';
+import { Toolbar } from 'components';
```

Comment:
Note with #2172, these need to be updated to prefix the dependencies with `@wordpress/`. You will need to perform a rebase against the latest version of master and apply your changes:

```
git fetch origin
git rebase origin/master
```